### ISSUE TITLE
Advanced random trigger does weird things to the advanced random trigger

### ISSUE THREAD

**Issue by Fnige** [SpaghettDev/BetterInputs#16]
Preemptively: I think this is similarish to #10 but still different so... 

For some reason, the advanced trigger's input fields acts weirdly with this mod. Seems to always follow the second field no matter what.

Without the mod:
![image](https://github.com/user-attachments/assets/72ccc44a-acf8-47b4-868d-5d417a26650a)
![image](https://github.com/user-attachments/assets/02251835-ed0f-4a65-811a-da74614f368b)

With the mod:
![image](https://github.com/user-attachments/assets/f2c16724-74e2-4991-ad41-43cad4ce3604)
![image](https://github.com/user-attachments/assets/c940592b-5b2d-4bc5-adc2-7ba38713c23f)

**Comment by SpaghettDev** [comment]
Yes it's the same issue, I'll close this one for the sake of not having duplicate issues.
